app/components/Menu.js

```diff
@@ -1,3 +1,5 @@
+import Link from "next/link";
```

Comment:
I am just adding a link here to prevent a hard refresh, so that nextjs will go navigate as an SPA